### PR TITLE
Touch ups

### PR DESCRIPTION
**Pull request by Q-Lee**
These are just a couple of corrections needed for the monitoring to deploy and function correctly.

### CODE REVIEW

File: prow/hook/server.go [hunk 1]
```diff
@@ -42,6 +42,11 @@ type Server struct {
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 
+	// Our health check uses GET, so just kick back a 200.
+	if r.Method == http.MethodGet {
```
Comment (rmmh) [inline prow/hook/server.go]: Can we specify that r.URL.Path == "/health" too?

File: prow/hook/server_test.go [hunk 2]
```diff
@@ -122,6 +122,18 @@ func TestServeHTTPErrors(t *testing.T) {
 			Body: body,
 			Code: http.StatusOK,
 		},
+		{
+			// Good
+			Method: http.MethodGet,
+			Header: map[string]string{
```
Comment (rmmh) [inline prow/hook/server_test.go]: these headers should be unnecessary

File: prow/hook/server.go (hunk 1 above)
Comment (Q-Lee) [reply prow/hook/server.go]: The "/hook" endpoint has special routing. I'd prefer to hit exactly this URL, so I don't need to worry about consistency.

File: prow/hook/server.go [hunk 3]
```diff
@@ -42,6 +42,11 @@ type Server struct {
...
+		return
```
[the remark this answers is not in the document]
Comment (Q-Lee) [reply prow/hook/server.go]: I could, but it would make hook a snowflake.

File: prow/hook/server_test.go (hunk 2 above)
Comment (Q-Lee) [reply prow/hook/server_test.go]: Duplicate

Comment (Q-Lee) [reply prow/hook/server_test.go]: Fair

**Pull request merged**